src/cryptoadvance/specter/process_controller/node_controller.py

```diff
@@ -231,8 +229,13 @@ def testcoin_faucet(self, address, amount=20):
             default_address = default_rpc.getaddressinfo(default_address)[
                 "unconfidential"
             ]
-        if balance < amount:
+        while True:
+            btc_balance = default_rpc.getbalance()
+            if isinstance(btc_balance, dict):  # elementsd
```

Comment:
This assumption is wrong. Our LiquidRPC class returns value only in Bitcoin unless you provide `asset=None` keyword.
See https://github.com/cryptoadvance/specter-desktop/blob/master/src/cryptoadvance/specter/liquid/rpc.py#L36-L53

Better to use `is_liquid(chain)` or check implementation from node info - Bitcoin Core or Elements Core

Reply:
I've wrote a small test which is currently failing. Maybe you can check:
https://cirrus-ci.com/task/6114508184748032?logs=test#L183